extensions/wikia/FlowTracking/scripts/createNewPageFlows.js

```diff
@@ -6,23 +6,31 @@ define(
 			articleId = mw.config.get('wgArticleId');
 
 		function trackOnEditPageLoad(editor) {
+			var qs = new QueryString(),
+				// 'flow' is the parameter passed in the url if user has started a flow already
+				flowParam = qs.getVal('flow', false);
+
 			// Track only creating articles (wgArticleId=0) from namespace 0 (Main)
 			// IMPORTANT: on Special:CreatePage even after providing article title the namespace is set to -1 (Special Page)
 			if (namespaceId === 0 && articleId === 0) {
-				var qs = new QueryString(),
-					// 'flow' is the parameter passed in the url if user has started a flow already
-					flowParam = qs.getVal('flow', false);
-
 				if (flowParam || document.referrer) {
-					//TODO: track middle step for other flows
+					flowTrack.trackFlowStep(flowParam, {editor: editor});
```

Comment:
so if we validate then we have 2 options:
a) don't track such cases
b) track that this is edited by user.

which approach in your opinion should be implemented?